src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -89,7 +89,6 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
     const policyID = policy?.id ?? '';
     // @ts-expect-error a new props will be added during feed api implementation
     const workspaceAccountID = policy?.workspaceAccountID ?? '';
-    // @ts-expect-error onyx key will be available after this PR https://github.com/Expensify/App/pull/44469
     const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARDS_LIST}${workspaceAccountID}_Expensify Card`);
```

Comment:
@VickyStash Let's put this `Expensify Card` into a const
```suggestion
    const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARDS_LIST}${workspaceAccountID}_Expensify Card`);
```

Reply:
@mountiny what has to be changed here? (I've changed EXPENSIFY for WORKSPACE already)